cli/cmd/tapRunner.go

```diff
@@ -131,21 +132,68 @@ func RunMizuTap() {
 		}
 		return
 	}
-	defer finishMizuExecution(kubernetesProvider)
+	if config.Config.Tap.DaemonMode {
+		err := handleDaemonModePostCreation(cancel, kubernetesProvider)
```

Comment:
change to one lines (if err := handleDaemonModePostCreation(cancel, kubernetesProvider); err != nil)